earthpy/spatial.py

```diff
@@ -112,7 +112,9 @@ def stack_raster_tifs(band_paths, out_path, arr_out=True):
     # Set default import to read
     kwds = {'mode': 'r'}
 
-    if not os.path.exists(os.path.dirname(out_path)):
+    out_dir = os.path.dirname(out_path)
+    writing_to_cwd = out_dir == ""
```

Comment:
@mbjoseph i have a hard time understanding this line. doesn't this set both vars to "" ? I know there is just something i dont understand here.

the `out_dir` parameter was previous `out_path` in hopes users would understand they need to provide a path to the directory not just the name of it. This was confusing for students

Reply:
Hi @lwasser - not quite: `out_dir` will be a string, and `writing_to_cwd` will be boolean because of the `==` operator.

```python
>>> out_dir = os.path.dirname("output.tif")
>>> out_dir
''
>>> writing_to_cwd = out_dir == ""
>>> writing_to_cwd
True
```

Reply:
Oh. i was reading this wrong. there is a double `==`. my brain has a hard time reading all of the equals in that line. i get it now. thank you.